var/spack/repos/builtin/packages/dla-future/package.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
+    """DLA-Future library: Distributed Linear Algebra with Future"""
+
+    homepage = "https://github.com/eth-cscs/DLA-Future"
+    url = "https://github.com/eth-cscs/DLA-Future/archive/v0.0.0.tar.gz"
+    git = "https://github.com/eth-cscs/DLA-Future.git"
+    maintainers = ["rasolca", "albestro", "msimberg", "aurianer"]
+
+    version("0.1.0", sha256="f7ffcde22edabb3dc24a624e2888f98829ee526da384cd752b2b271c731ca9b1")
+    version("develop", branch="master")
+
+    cxxstds = ("17", "20")
+    variant(
+        "cxxstd",
+        default="17",
+        values=cxxstds,
+        description="Use the specified C++ standard when building",
+    )
```

Comment:
Since this doesn't control anything directly in DLA-Future, can't we leave this out? Instead of saying `dla-future cxxstd=17` I should just say `dla-future ^pika cxxstd=17`, no?

If you want to keep the conflict with C++20 and CUDA we can instead have `conflicts("pika cxxstd=20", when="+cuda")`?

Reply:
https://github.com/spack/spack/pull/38224/commits/196d8ae0c6bfdf26da18ba838fadef4fd40b2b38